doc/short_vignettes/selective_sweep.md

```diff
@@ -0,0 +1,152 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(selective_sweeps)=
+
+# Selective sweeps
+
+```{code-cell} python
+---
+tags: ['hide-input']
+---
+
+import fwdpy11
+import numpy as np
+import msprime
+```
+
+```{code-cell} python
+import fwdpy11.conditional_models
+import fwdpy11.tskit_tools
+```
+
+
+```{code-cell} python
+---
+tags: ['hide-input']
+---
+
+def setup(prune_selected=False):
+    # Dropping mutations requires existing
+    # ancestry, which we can get either
+    # from a burn-in or from msprime.
+    initial_ts = msprime.sim_ancestry(
+        samples=500,
```

Comment:
Do you want `samples` to equal `population_size`?

Reply:
I need to  double-check this.